geoapi/tests/api_tests/test_streetview_routes.py

```diff
@@ -118,45 +142,66 @@ def test_delete_organization(test_client, organization_fixture):
-
+    resp = test_client.delete('streetview/instances/{}/'.format(instance_fixture.id),
+                              headers={'x-jwt-assertion-test': u1.jwt})
     assert resp.status_code == 200
-    assert len(data['sequences']) == 4
+    assert db_session.query(StreetviewInstance).first() is None
 
 
-def FAILING_test_delete_streetview_sequence(test_client):
+def test_add_sequence_to_instance(test_client,
+                                  streetview_service_resource_fixture,
+                                  organization_fixture):
     u1 = db_session.query(User).get(1)
     data = {
-        'dir': {
-            'path': 'test path',
-            'system': 'test system'
-        },
-        'service': 'mapillary',
-        'sequences': ['test key 1', 'test key 2', 'test key 3', 'test key 4']
+        "streetviewId": streetview_service_resource_fixture.id,
+        "sequenceId": "my_sequence_id",
+        "organizationId": "my_org_id",
+        "dir": {
+            "system": "my_system",
+            "path": "my_path"
+        }
     }
+    resp = test_client.post('/streetview/sequences/', json=data, headers={'x-jwt-assertion-test': u1.jwt})
+    assert resp.status_code == 200
+    # NOTE: shouldn't response by the instance
 
-    resp = test_client.post('/streetview/sequences/',
-                            json=data,
-                            headers={'x-jwt-assertion-test': u1.jwt})
+    streetview_instance = db_session.query(StreetviewInstance).first()
+    assert len(streetview_instance.sequences) == 1
+    sequence = streetview_instance.sequences[0]
+    assert sequence.sequence_id == "my_sequence_id"
+    assert sequence.organization_id == "my_org_id"
+    assert sequence.streetview_instance_id == streetview_service_resource_fixture.id
 
-    resp = test_client.get('/streetview/',
+
+def test_get_sequence(test_client, streetview_service_resource_fixture, sequence_fixture):
+    u1 = db_session.query(User).get(1)
+    resp = test_client.get('/streetview/sequences/{}/'.format(sequence_fixture.sequence_id),
                            headers={'x-jwt-assertion-test': u1.jwt})
+    assert resp.status_code == 200
+    assert resp.get_json() == {'bbox': None, 'end_date': None, 'id': 1, 'organization_id': '1',
+                               'sequence_id': sequence_fixture.sequence_id, 'start_date': None, 'streetview_instance_id': 1}
 
-    seq_id = resp.get_json()[0]['sequences'][0]['id']
 
-    test_client.delete('/streetview/sequences/{}/'.format(seq_id),
-                       headers={'x-jwt-assertion-test': u1.jwt})
+def test_update_sequence(test_client, streetview_service_resource_fixture, sequence_fixture):
+    u1 = db_session.query(User).get(1)
+    data = {
+        "organization_id": "something_else",
+    }
+    resp = test_client.put(
+        '/streetview/sequences/{}/'.format(sequence_fixture.id),  # TODO differs from GET route where mapillary id is used
```

Comment:
@duckonomy  Note that here (PUT) and in the DELETE the route is expecting the id (int) and not the sequence_id (uid string from mapillary).  I assume it all works as is on frontend but we should note to maybe alter this later.